### DISCUSSION TITLE
Popp 701721 radiator thermostats TRV

### DISCUSSION THREAD

**Discussion by Mikkooooo** [Koenkk/zigbee2mqtt#19733] [Devices]
Hi,
I've added 2 Popp smart TRV's   701721 .
1.     Sometimes it takes many attempts to add them.
2.     After adding I wanted to update the devices and then there comes an error message   "No image available for image type '288'  "
 
What's going wrong ?

**Comment by lux73** [comment]
answer for 2. - there is simple no update for your Device available ^^

**Reply by Mikkooooo** [reply]
Thanks , I’m wondering there comes a red colored message  „Failed to check update“      AND       "No image available for image type '288'
